package.json

```diff
@@ -91,7 +92,7 @@
     "scuttle-thread": "^1.0.1",
     "setimmediate": "^1.0.5",
     "ssb-about": "^2.0.1",
-    "ssb-ahoy": "github:ssbc/ssb-ahoy#add_config_picker",
+    "ssb-ahoy": "github:ssbc/ssb-ahoy#electron_4",
```

Comment:
this version of `ssb_ahoy` is the same as `add_config_picker` branch except:
- it uses electron 4.2.0
- it has no `cross-script` / `electron-builder` (not quite sure what that will do)

Reply:
Sweet, I think as long as the ssb-ahoy Electron ABI matches the Patchbay Electron ABI then this should work great. No `electron-builder install-app-deps` necessary since Patchbay rebuilds all dependencies to match its Electron ABI.